modules/agent-install-about-mirroring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// * installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc
+// re-use of applicable content from disconnected install mirroring
+
+:_content-type: CONCEPT
+[id="agent-install-about-mirroring-for-disconnected-registry_{context}"]
+= About mirroring the {product-title} image repository for a disconnected registry
+
+To use mirror images for a disconnected installation through the Agent-based Installer, you must modify the `install-config.yaml` file.
+
+You can mirror the release image by using the output of either the `oc adm release mirror` or `oc mirror` command.
+This is dependent on which command you used to set up the mirror registry. The following example shows the output of the `oc adm release mirror` command.
+
+
+[source,terminal]
+----
+$ oc adm release mirror
+----
+
+.Example output
+
+[source,terminal]
+----
+To use the new mirrored repository to install, add the following
+section to the install-config.yaml:
+
+imageContentSources:
+
+mirrors:
+virthost.ostest.test.metalkube.org:5000/localimages/local- release-image
+source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+mirrors:
+virthost.ostest.test.metalkube.org:5000/localimages/local- release-image
+source: registry.ci.openshift.org/ocp/release
+----
+Use the output of the command to the `imageContentSources` field in the `install-config.yaml` file. Additionally add the certificate file used for the mirror registry to the `additionalTrustBundle` field.
```

Comment:
Maybe "Use the output of the command to update the imageContentSources field..."